scripts/ebpf_tracing.cmd

```diff
@@ -1,18 +1,6 @@
 @rem Copyright (c) Microsoft Corporation
 @rem SPDX-License-Identifier: MIT
 
-@rem Script behavior:
-@rem - When called with 'start', it will:
```

Comment:
Why are the comments deleted?
This seems like a regression and unrelated to the point of the PR.

Reply:
@gtrevi still waiting for an update or answer

Reply:
@dthaler, the comments were redundant with the actual command description.